packages/perseus/src/widgets/interactive-graphs/stateful-mafs-graph.tsx

```diff
@@ -128,5 +129,15 @@ export const StatefulMafsGraph = React.forwardRef<
         graph.startCoords,
     ]);
 
+    if (props.static) {
```

Comment:
This becomes obvious fairly quickly as I read the code, but could you leave a comment here that if we're in `static` mode we display the correct answer? (unless you don't feel that's necessary)